cockatrice/src/tab.h

```diff
@@ -33,6 +33,7 @@ protected slots:
     virtual void retranslateUi() = 0;
     virtual void closeRequest() { }
     virtual void tabActivated() { }
+    virtual bool updateIcon() { return true; }
```

Comment:
I think I'm going to disable the "unused function" lint since it tends to have many false positives, especially with Qt's signal/slots system